source/pages/skymap_page/Constellation.js

```diff
@@ -47,40 +47,48 @@ export class Constellation {
     }
   }
 
-  /*
-    Click event handler, pass click event to all stars in the constellation
-    */
+  /**
+   * Click event handler, pass click event to all stars in the constellation
+   * @param {Number} x
+   * @param {Number} y
+   */
   click(x, y) {
     this.stars.forEach((star) =>
       star.click(x - this.offset_x, y - this.offset_y)
     );
   }
 
-  /*
-    Draw function, draw all stars in constellation
-    */
+  /**
+   * Draw function, draw all stars in constellation
+   * @param {Number} user_x user view offset x
+   * @param {Number} user_y user view offset y
+   */
   draw(user_x, user_y) {
     this.stars.forEach((star) => star.update(user_x, user_y));
   }
 
-  /*
-    Update function, to be called by animation loop
-    */
+  /**
+   * Update function, to be called by animation loop
+   * @param {Number} user_x user view offset x
+   * @param {Number} user_y user view offset y
+   */
   update(user_x, user_y) {
     this.draw(user_x, user_y);
   }
 
-  /*
-    New update function if the current constellation is the final result.
-    */
+  /**
+   * New update function if the current constellation is the final result.
+   * @param {Number} user_x user view offset x
+   * @param {Number} user_y user view offset y
+   */
   updateNew(user_x, user_y) {
     this.draw(user_x, user_y);
     this.connectAll();
   }
 
-  /*
-        Connect all the points in the constellation
-        */
+  /**
+   * Connect all the points in the constellation
+   */
```

Comment:
check #132 but just add a `@property {Function} description of function` if there's no parameters or return for the function